mlflow/requirements.txt

```diff
@@ -0,0 +1,11 @@
+mlflow
```

Comment:
Is `mlflow` along okay? Or do we need `mlflow[genai]`? See https://mlflow.org/docs/latest/llms/deployments/index.html